README.md

```diff
@@ -529,6 +530,30 @@ OPTIONS
 
 _See code: [src/commands/es/indices/insert.ts](src/commands/es/indices/insert.ts)_
 
+## `kourou es:migrate`
+
+Migrate all the index from one Elasticsearch server to another
```

Comment:
```suggestion
Migrate all the indexes from one Elasticsearch server to another
```

Reply:
I'll update it in the migrate.ts file and generate a new README